packages/host/app/components/operator-mode/code-submode.gts

```diff
@@ -916,32 +954,7 @@ export default class CodeSubmode extends Component<Signature> {
                       @cardTypeResource={{this.selectedCardOrField.cardType}}
                       @openDefinition={{this.openDefinition}}
                     />
-                  {{else if this.schemaEditorIncompatibleFile}}
-                    <div
-                      class='incompatible-schema-editor'
-                      data-test-schema-editor-incompatible-file
-                    >
-                      Schema Editor cannot be used with this file type.
-                    </div>
-                  {{else if
-                    (and
-                      this.isValidSchemaFile this.schemaEditorIncompatibleItem
-                    )
-                  }}
-                    <div
-                      class='incompatible-schema-editor'
-                      data-test-schema-editor-incompatible-item
-                    >
-                      Schema Editor cannot be used for selected
-                      {{this.selectedDeclaration.type}}
-                      "{{this.selectedDeclaration.localName}}".</div>
-                  {{else if this.cardError}}
-                    {{this.cardError.message}}
                   {{/if}}
-                {{else if this.isLoading}}
-                  <div class='loading'>
-                    <LoadingIndicator />
-                  </div>
```

Comment:
These conditional templates were kind of confusing. I realise that all the errors shown actually just mean that NOTHING on the rhs can show it. So I coupled everything into `rhsIncompatibleMessage`